.github/workflows/docker.yml

```diff
@@ -52,15 +53,15 @@ jobs:
         run: docker build . --file ./Dockerfile --tag ghcr.io/eol-uchile/edx-staging:s3-static-${{ env.CODE }}-${{ env.RELEASE-NAME }} --tag ghcr.io/eol-uchile/edx-staging:s3-static-${{ env.CODE }}-${{ env.RELEASE-NAME }}-${{ env.TS }} --target s3
 
       - name: Login to GitHub Container Registry
-        if: github.ref == format('refs/heads/{0}-release/{1}', ${{ env.PROJECT }}, ${{ env.RELEASE-NAME }})
+        if: github.ref == format('refs/heads/{0}-release/{1}', ${{ env.PROJECT }}, ${{ env.RELEASE-NAME }}) && ${{ env.IS-PR == 'false' }}
```

Comment:
Ya sabemos que el if no estaba funcionando, pero de todos modos la idea era reemplazar el test previo con la condición is-pr, porque es equivalente. Que github.ref apunte a la rama release, sólo puede significar que no es un pr ni una mq. Ojo ahí, si el if hubiera funcionado (creo que no lo hace por un tema de sintaxis), estarías agregando una redundancia.

Reply:
done;